section4/exercises/objects.js

```diff
@@ -96,12 +112,38 @@ console.log(posts);
 console.log(posts[0]);
 
 /*
-The code snippet above shows an Array with 2 elements. Each element in an 
-Object Literal. Each of those Object Literals has 4 key-value pairs. This may LOOK 
+The code snippet above shows an Array with 2 elements. Each element in an
+Object Literal. Each of those Object Literals has 4 key-value pairs. This may LOOK
 a bit daunting - it's OK! You don't need to be 100% comfortable with this, but it's
 good to have some exposure before going into Mod 1.
 */
 
-// YOU DO: Create an array of at least 3 EMAIL Object Literals, using the same 
+// YOU DO: Create an array of at least 3 EMAIL Object Literals, using the same
 // key-value pairs you used in your email Object above.
-// Then, log the email Array to the console.
\ No newline at end of file
+// Then, log the email Array to the console.
+
+var emails = [
+  {
+    sender: 'Amy Holt',
+    recipient: 'Rachel Hill',
+    subject: 'Mod 1 Prework Feedback',
+    timeStamp: '9:30 PM December 13, 2021',
+    wordCount: 123
+  },
+  {
+    sender: 'Rachel Hill',
+    recipient: 'Amy Holt',
+    subject: 'Mod 0 Assessment Questions',
+    timeStamp: '2:30 PM December 14, 2021',
+    wordCount: 15
+  },
+  {
+    sender: 'Margot S.',
+    recipient: 'Mod 0 2111a Cohort',
+    subject: 'Reminder: Turn in Mod 1 Prework!',
+    timeStamp: '10:00 AM December 17, 2021',
+    wordCount: 35
+  }
+]
+
```

Comment:
Nice work with objects! This example where you have objects nested inside of an array (a list of objects) is something you'll see _a lot_ as you progress through Turing 🙂